frontend/src/app/testQueue/TestQueue.test.tsx

```diff
@@ -1,30 +1,45 @@
-import { render, waitFor } from "@testing-library/react";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
 import { MockedProvider } from "@apollo/client/testing";
 import TestQueue, { queueQuery } from "./TestQueue";
+import { REMOVE_PATIENT_FROM_QUEUE } from "./QueueItem";
 import { QUERY_PATIENT } from "./addToQueue/AddToQueueSearch";
 
 jest.mock("@microsoft/applicationinsights-react-js", () => {
   return {
     useAppInsightsContext: jest.fn(),
-    useTrackEvent: jest.fn(),
+    useTrackEvent: () => jest.fn(),
   };
 });
 
 describe("TestQueue", () => {
   it("should render the test queue", async () => {
-    const { container, getByText, getByLabelText } = render(
+    const { container } = render(
       <MockedProvider mocks={mocks}>
         <TestQueue activeFacilityId="a1" />
       </MockedProvider>
     );
-
-    await waitFor(() => getByLabelText("Search"));
-    await waitFor(() => new Promise((res) => setTimeout(res, 0)));
-
-    expect(getByText("Doe, John A")).toBeInTheDocument();
-    expect(getByText("Smith, Jane")).toBeInTheDocument();
+    await screen.findByLabelText("Search");
+    expect(await screen.findByText("Doe, John A")).toBeInTheDocument();
+    expect(await screen.findByText("Smith, Jane")).toBeInTheDocument();
     expect(container).toMatchSnapshot();
   });
+  it("should remove items queue using the transition group", async () => {
```

Comment:
@dmethvin-gov do you have any time to help me figure out why I can't get this test to pass? I'd like to get at least one test working for this functionality.